atlp-rwanda/space-bn-frontend
# fix(#177033396) :  User should confirm email before accessing dashboard

### **What does this PR do**
- It added a chore to request for email verification before direct access to dashboard

### **How should this be tested**
- clone the repository
- run `npm install`
- switch to brache `fix-confirm-email-#177033396`
- run `npm start` 
- signup a new user

### **Expected output**
- You should be prompted with `verify-email` page

### **Any background information you want to provide**
- If one logs in without verifying email the same page will be prompted
- In case the verification email was not received the bottom link can be used to resend the verification

### **Relevant Pivotal tracker story**
- #177033396

### **Any background image**
N/A
